### ISSUE TITLE
EF Core 3 issue?

### ISSUE THREAD

**Issue by VictorioBerra** [JonPSmith/EfCore.GenericServices#36]
Got:

```
System.MissingMethodException: 'Method not found: 'Boolean Microsoft.EntityFrameworkCore.Metadata.IEntityType.get_IsQueryType()'.'
```

when doing:

```
services.GenericServicesSimpleSetup<ApplicationDbContext>(Assembly.GetAssembly(typeof(TestDto)));
```

**Comment by VictorioBerra** [comment]
I don't see that method here https://github.com/aspnet/EntityFrameworkCore/blob/release/3.1/src/EFCore/Metadata/IEntityType.cs

**Comment by JonPSmith** [comment]
I am currently working on GenericServices to supports NET Core +=2.1 and Net Core >=3.0. I will let you know when I have a version.

**Comment by VictorioBerra** [comment]
Sounds great, thanks Jon.

**Comment by JonPSmith** [comment]
I have just released version 3.0.0 which supports EF Core >=3.0 (and still supports EF Core >=2.1).

**Comment by VictorioBerra** [comment]
Great work!! Will try shortly!
